contracts/SpokePool.sol

```diff
@@ -1105,37 +1105,31 @@ abstract contract SpokePool is
 
     // The following internal methods emit events with many params to overcome solidity stack too deep issues.
     function _emitFillRelay(RelayExecution memory relayExecution, uint256 fillAmountPreFees) internal {
-        RelayExecutionInfo memory relayExecutionInfo;
-        // This separate struct is created entirely to avoid stack too deep errors.
-        {
-            relayExecutionInfo = RelayExecutionInfo({
-                relayerFeePct: relayExecution.updatedRelayerFeePct,
-                recipient: relayExecution.updatedRecipient,
-                message: relayExecution.updatedMessage,
-                isSlowRelay: relayExecution.slowFill,
-                payoutAdjustmentPct: relayExecution.payoutAdjustmentPct
-            });
-        }
+        RelayExecutionInfo memory relayExecutionInfo = RelayExecutionInfo({
```

Comment:
Great call -- I figured these would be necessary even with viaIR, but I was wrong!